### DISCUSSION TITLE
Connection to HM-600 lost after firmware upgrade to b72a98d

### DISCUSSION THREAD

**Discussion by andreasschueler** [tbnobody/OpenDTU#480] [Q&A]
After firmware upgrade to opendtu-generic.bin i no data is received from my HM-600. 

Here is what i get from console frequently, i re-added the HM-600 via it's serial number and also rebooted OpenDTU and the Hoymiles.

Any hint ?

Interrupt received

RX Period End
All missing
Nothing received, resend count exeeded
Interrupt received
TX AlarmData Channel: 23 --> 15 83 09 28 65 80 14 61 68 80 11 00 63 B0 17 30 00 00 00 00 00 00 00 00 18 8C BE

**Comment by tbnobody** [comment]
> Interrupt received

This sounds good as the NRF module is receiving data from the inverter... But it's strange that there is no parsing after this line, I would have expected a RX line after this.
